src/main/java/com/rackspacecloud/metrics/ingestionservice/influxdb/InfluxDBHelper.java

```diff
@@ -214,39 +205,34 @@ private boolean databaseExists(final String databaseName, final String baseUrl)
             return false;
```

Comment:
This seems like a spot where an exception should be thrown since a `false` does not correctly reflect the fact that the query failed before the existence could be determined.